source/shared/xplat.h

```diff
@@ -234,162 +117,48 @@ typedef windowsULongLong_t ULONGLONG;
 // source code shared with Windows SNAC (which is ODBC 3.8) compiles.
 #define ODBCVER 0x0380
```

Comment:
I believe in appveyor it configures without that odbcver option already, so we will see